gpiozero/spi_devices.py

```diff
@@ -92,9 +131,9 @@ def _read(self):
     def value(self):
         """
         The current value read from the device, scaled to a value between 0 and
-        1.
+        1 (or -1 to +1 for certain devices operating in differential mode).
```

Comment:
If this can return **either** a 0->1 value or a -1 -> +1 value, does that imply that this class itself should 'know' if it's operating in differential mode, and offer that as a property?